src/util/support.js

```diff
@@ -1,5 +1,38 @@
+import isNativeRegex from 'regex-native-function'; // checks if a function is native using regex
+
 const div = document.createElement('div');
+const isNative = (elem, prop) => isNativeRegex.test(elem[prop]);
+const isPolyfill = (elem, prop) => !!elem[prop] && !isNative(elem, prop);
+
+export const v0ShadowDOMProperty = 'createShadowRoot'; // property used to check for shadowDOMv0 support
```

Comment:
nit: comments go above associated lines, empty newline should separate comment and code above it, i.e.:

```js
// comment 1
const code1;

// comment 2
const code2;
```

The aim is readability and comment association.